spec/controllers/users_spec.rb

```diff
@@ -1,8 +1,13 @@
-# require 'rails_helper'
+require 'rails_helper'
 
 RSpec.describe 'Users', type: :request do
+  before(:each) do
+    @user = User.create(name: 'Tadesse Jemal', photo: 'https://unsplash.com/photos/mateopic',
+                        bio: 'Software Developer, Ethiopian, I like to cook', posts_counter: 5)
+  end
+
```

Comment:
- [ ] Good job adding some tests but please kindly extract the User index tests from the show tests into separate files. Also, the following tests are missing

### Index Page Tests
* Tests that assert you can see the username of all other users.
* Tests that assert you can see the profile picture for each user.
* Tests that assert you can see the number of posts each user has written.
* Tests that redirect to the user's show page when you click on a user.

### Show Page Tests
* Tests that asserts you can see the user's profile picture.
* Tests that asserts you can see the user's username.
* Tests that asserts you can see the number of posts the user has written.
* Tests that asserts you can see the user's bio.
* Tests that asserts you can see the user's first 3 posts.
* Tests that asserts you can see a button that lets me view all of a user's posts.
* Tests that redirects to the post's show page when you click on a user's post.
* Tests that redirects to the user's post's index page when you click on a **see all posts**.

Reply:
Hello reviewer, thank you for the feedback, sorry to say this but you didn't check that we have created the test. 
![image](https://user-images.githubusercontent.com/85793292/206030318-48f0e031-e6e5-4d10-a5b4-6b0c44c387f9.png)
please check it again.

Reply:
hi, dear review, I realize that you checked the wrong test because I created special tests for the view, please check it inside the view folder.